lxd/device/nic_bridged.go

```diff
@@ -214,6 +215,10 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 		var err error
 		d.network, err = network.LoadByName(d.state, api.ProjectDefaultName, d.config["network"])
 		if err != nil {
+			if d.config["required"] == "false" {
```

Comment:
I believe the intention with this PR was to allow instance device to start if the parent network was not in the "up" state.

I still think it should fail to start if it references an unknown managed network - this is a configuration issue rather than an environmental issue.